packages/redux-routine/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 3.0.4
```

Comment:
Should include `(Unreleased)` when not yet published

https://github.com/WordPress/gutenberg/blob/master/CONTRIBUTING.md#maintaining-changelogs

```suggestion
## 3.0.4 (Unreleased)
```